test/e2e/network_connect.go

```diff
@@ -0,0 +1,1898 @@
+package e2e
```

Comment:
I have just realized we have a file called `cluster_network_connect_validations.go` does it mean this one should be called `cluster_network_connect.go` then?